libdrizzle/pack.cc

```diff
@@ -481,3 +481,24 @@ static drizzle_return_t _pack_scramble_hash(drizzle_st *con,
 
   return DRIZZLE_RETURN_OK;
 }
+
+void drizzle_unpack_error(drizzle_st *con)
```

Comment:
Does it make sense to call this function from https://github.com/sociomantic-tsunami/libdrizzle-redux/blob/c07059be4eb5027c593391848165141c4ef4a655/libdrizzle/result.cc#L467-L481 to avoid code duplication?